www/src/pages/en/deployment/vercel.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Vercel CLI?

Reply:
I think deploying from the github repository is fine

Reply:
IMO anyone who wants to use the Vercel CLI probably doesn't need our docs for it

Reply:
Fair point 👍